packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md

```diff
@@ -0,0 +1,262 @@
+     --data 'grant_type=client_credentials&scope=okta.users.read okta.groups.read'
+   ```
+
+1. If the request is successful, the token is returned in the body of the response.
+
+   Successful response example:
+
+   ```json
+   {
+      "token_type": "Bearer",
+      "expires_in": 3600,
+      "access_token": "eyJraWQiOiJ.....UfThlJ7w",
+      "scope": "okta.users.read okta.groups.read"
+   }
+   ```
+
+   Error response example:
+
+   ```json
+   {
+      "error": "invalid_client",
+      "error_description": "The client secret supplied for a confidential client is invalid."
+   }
+   ```
+
+   See the [list of token endpoint errors](/docs/reference/api/oidc/#possible-errors-4).
+
+> **Note:** By default, the access token expires in an hour. You need to request for a new access token after your token expires.
+
+### Make Okta API requests
+
+When you have an access token, you can use it to make requests to the [Okta management APIs](/docs/reference/core-okta-api/). Set the access token as a bearer token in an authorization header.
+
+Example request:
+
+```bash
+curl -X GET "https://${customerOktaDomain}/api/v1/users"
+    -H "Accept: application/json"
+    -H "Authorization: Bearer ${accessToken}"
+```
+
+### Test your API service flow
+
+Use the credentials from your test integration (Okta domain, client ID and secret) to test your service app. You can use the [interface that you build](#collect-and-save-customer-credentials) or enter the credentials manually to configure your service app. Trigger a process from your service app to access Okta endpoints and verify that the process works as intended.
+
+After you completed testing and are satisfied with your API service integration, you can [submit it to Okta](#submit-for-review) for verification.
+
+### Edit your API service integration submission
+
+To edit your API service integration submission fields:
+
+1. From the [OIN Manager](https://oinmanager.okta.com/), click **Edit** next to your API service integration submission.
+1. Edit the fields you want to modify.
+
+The fields are automatically saved in your submission.
+
+If the fields that you update affect the test integration instance in your Okta org (such as integration name or scopes), then you must click **Test in Okta** to create another test integration instance with the updated fields. The test integration instance is a static snapshot of your integration submission fields at the time it was created. It doesn't get updated when you modify your submission. If you want to remove older test integration instances that aren't used, see [Remove your test integration instance](#remove-your-test-integration-instance).
+
+### Remove your test integration instance
+
+To remove the test API service integration in your Okta org, click **Revoke** from the integration details page in your Okta Admin Console. This action revokes the scope grants, client ID and secret from the test integration that you installed previously.
```

Comment:
The section above implies you can create several test integrations, so "a" seems more appropriate than "the"
```suggestion
To remove a test API service integration in your Okta org, click **Revoke** from the integration details page in your Okta Admin Console. This action revokes the scope grants, client ID, and secret from the test integration installed previously.
```